pom.xml

```diff
@@ -45,6 +45,7 @@
         <gridsuite-dependencies.version>35</gridsuite-dependencies.version>
         <string-template.version>4.3.1</string-template.version>
         <liquibase-hibernate-package>org.gridsuite.filter.server</liquibase-hibernate-package>
+        <sonar.coverage.exclusions>**/migrations/**/*</sonar.coverage.exclusions>
```

Comment:
We could try to find another solution for the location of the migration scripts. Not in this PR if it's not so easy

Reply:
ok

Reply:
Finally we keep it like this